### PR TITLE
Log aleth output upon test failure

### PR DESCRIPTION
**Pull request by Marenz**
refs #5803

### CODE REVIEW

**Comment by Marenz** [comment]
By now I actually believe the problem is in aleth.

**Comment by chriseth** [comment]
Is this still neeed?

**Comment by Marenz** [comment]
well.. it wouldn't hurt, but I a working on another PR that will provide us with alteh logs in the case of an error

**Comment by Marenz** [comment]
Changed PR to log aleth output now

File: scripts/tests.sh [hunk 1]
```diff
@@ -149,7 +152,7 @@ function download_aleth()
 # echos the PID
 function run_aleth()
 {
-    $ALETH_PATH --db memorydb --test -d "${WORKDIR}" >/dev/null 2>&1 &
+    $ALETH_PATH --log-verbosity 3 --db memorydb --test -d "${WORKDIR}" > $ALETH_TMP_OUT 2>&1 &
```
Comment (chriseth) [inline scripts/tests.sh]:
```suggestion
    $ALETH_PATH --log-verbosity 3 --db memorydb --test -d "${WORKDIR}" > "$ALETH_TMP_OUT" 2>&1 &
```

**Comment by chriseth** [comment]
Could you make the log an artifact (like the test result xml files) so it can be downloaded after the CI Run?

**Comment by Marenz** [comment]
@chriseth that's exactly what was intended, it just didn't work because I have made a mistake somewhere. Maybe it will work this time

**Review by chriseth — APPROVED** [review]

**Pull request merged**